app/controllers/SubscriptionConfigurationController.scala

```diff
@@ -52,9 +52,9 @@ class SubscriptionConfigurationController @Inject()(val applicationService: Appl
 
   def listConfigurations(appId: ApplicationId): Action[AnyContent] = requiresAtLeast(GatekeeperRole.SUPERUSER) {
     implicit request =>
-        withAppAndFieldDefinitions(appId) {
+        withAppAndSubscriptionsAndFieldDefinitions(appId) {
```

Comment:
Can we just rename this withKitchenSink ?

Reply:
If you like